packages/graphql/src/translate/create-auth-and-params.ts

```diff
@@ -61,8 +61,53 @@ export function createAuthAndParams({
     bind?: Bind;
     where?: { varName: string; chainStr?: string; node: Node };
 }): [string, any] {
```

Comment:
Does this actually return `any`, or is it an object that can be typed? I'm just looking at the early return added with if `(!authPredicate) return ["", {}];`